src/py42/clients/securitydata.py

```diff
@@ -172,19 +172,38 @@ def get_all_user_security_events(
         )
 
     def search_file_events(self, query):
-        """Searches for file events.
+        """Searches for file events, returns up to the first 10,000 events.
         `REST Documentation <https://support.code42.com/Administrator/Cloud/Monitoring_and_managing/Forensic_File_Search_API>`__
 
         Args:
-            query (:class:`py42.sdk.queries.fileevents.file_event_query.FileEventQuery`): Also
-                accepts a raw JSON str.
+            query query (`:class:`py42.sdk.queries.fileevents.file_event_query.FileEventQuery` or str): The file event query to filter search results.
 
         Returns:
             :class:`py42.response.Py42Response`: A response containing the first 10,000
             events.
         """
         return self._file_event_service.search(query)
 
+    def search_all_file_events(self, query, page_token=""):
+        """Searches for all file events, returning a page of response with a token in the response to retrieve next page.
+
+        `REST Documentation <https://support.code42.com/Administrator/Cloud/Monitoring_and_managing/Forensic_File_Search_API>`__
+
+        Args:
+            query query (`:class:`py42.sdk.queries.fileevents.file_event_query.FileEventQuery` or str): The file event query
+                to filter search results.
+            page_token (str): A token used to indicate the starting point for additional page results. Defaults to empty string.
```

Comment:
This should be `str or None` It should default to None

Reply:
You should not default to empty string in python or weird things can happen, instead default to `None` and set it to emptry string if it's `None` within the method.